dataplane/internal/engine/table.go

```diff
@@ -446,7 +441,7 @@ func AddLayer3PuntRule(ctx context.Context, c fwdpb.ServiceClient, portName stri
 }
 
 // AddIPRoute adds a route to the FIB.
-func AddIPRoute(ctx context.Context, c fwdpb.ServiceClient, v4 bool, ip, mask, nextHopIP []byte, port string) error {
+func AddIPRoute(ctx context.Context, c fwdpb.ServiceClient, v4 bool, ip, mask, nextHopIP []byte, port string, extraActs []*fwdpb.ActionDesc) error {
```

Comment:
Add comment for extra actions. Maybe name them pre-transmit actions as well? Or post-look-up actions?